images/scripts/fedora-32.bootstrap

```diff
@@ -18,4 +18,4 @@
 # 02110-1301 USA.
 
 BASE=$(dirname $0)
-$BASE/virt-install-fedora "$1" x86_64 "https://dl.fedoraproject.org/pub/fedora/linux/releases/32/Server/x86_64/os/"
+$BASE/virt-install-fedora "$1" x86_64 "https://dl.fedoraproject.org/pub/fedora/linux/releases/33/Server/x86_64/os/"
```

Comment:
Err, no -- you need to adjust the fedora-testing.* symlinks :grin:

Reply:
lol.  or read a patch before pushing it :rofl: